extensions/sensors/SensorLoader.cpp

```diff
@@ -0,0 +1,30 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "SensorLoader.h"
+
+#include "core/FlowConfiguration.h"
+
+bool SensorFactory::added = core::FlowConfiguration::add_static_func("createSensorFactory");
+
+extern "C" {
+
+void *createSensorFactory(void) {
+  return new SensorFactory();
```

Comment:
Maybe return a moved unique_ptr to make ownership semantics clearer?

Reply:
Can't really do that here due to where this could be called in C code.